gui/common/freenasldap.py

```diff
@@ -1893,11 +1893,16 @@ def get_netbios_name(self):
             basedn
 
         netbios_name = None
+        smb = Client().call('smb.config')
```

Comment:
Using Client() like this is wrong as it doesnt close the connection.

```
from freenasUI.middlewared.client import client
with client as c:
    c.call('smb.config')
```

we should probably fix the other usages in this file.